tests/integration/backend_test.py

```diff
@@ -705,6 +708,7 @@ def test_no_opt(authenticated_quum_backend: QuantinuumBackend) -> None:
     "authenticated_quum_backend", [{"device_name": "H1-1SC"}], indirect=True
 )
 def test_qir_submission(authenticated_quum_backend: QuantinuumBackend) -> None:
+    gc.disable()
```

Comment:
Should probably put a comment explaining why this is here (and maybe a low-priority issue to find the root cause).

Reply:
done in @cqc-melf
[add comment about gc](https://github.com/CQCL/pytket-quantinuum/pull/169/commits/77f37e62b9576176695334735ed252718347011d)
[77f37e6](https://github.com/CQCL/pytket-quantinuum/pull/169/commits/77f37e62b9576176695334735ed252718347011d)